cms/djangoapps/contentstore/features/component.py

```diff
@@ -74,8 +74,7 @@ def see_a_problem_component(step, category):
                 'No problem was added to the unit.')
 
     problem_css = 'li.studio-xblock-wrapper div.xblock-student_view'
-    actual_text = world.css_text(problem_css)
-    assert_in(category.upper(), actual_text)
+    assert_true(world.css_contains_text(problem_css, category.upper()))  # pylint: disable=no-value-for-parameter
```

Comment:
Another question... I know it was there before, but do you know why `.upper()` is used here? Is it making an assumption that the text in the problem is always uppercase?